src/dreammaker/config.rs

```diff
@@ -74,6 +83,10 @@ impl Config {
     pub fn registerable_error(&self, error: &DMError) -> bool {
         self.display.error_level.applies_to(error.severity())
     }
+
+    pub fn code_standards(&self) -> &CodeStandards {
+        &self.code_standards
```

Comment:
It's probably fine if this is a `pub` field but it's not that important either way